### ISSUE TITLE
r.db("rethinkdb").table("jobs") return wrong value if the query use between

### ISSUE THREAD

**Issue by ywtsang** [rethinkdb/rethinkdb#6240]
this problem happen for our new installed rethinkdb server with version 2.2.6~0trusty while it is ok in 2.1

when executing long running query using `between`, we can check the `jobs` table `r.db("rethinkdb").table("jobs")` to see the query but the `lowerkey` and `upperkey` are swapped

**Comment by AtnNn** [comment]
Thanks for the report. I can confirm this happens in 2.3.5

```
> r.db('rethinkdb').table('jobs').union(r.table('foo').between(1, 2))('info')('query')
'r.db("rethinkdb").table("jobs").union(r.table("foo").between(2, 1))("info")("query")'
```

**Comment by AtnNn** [comment]
This affects more than just `between`. The pretty-printer, in `visit_stringing`, pushes arguments in their natural order onto a stack of tokens that gets reversed (e.g. `[")", "1", "2", "(", "between"]`). This code should not be so complicated.

**Comment by AtnNn** [comment]
Fixed by #6245

**Comment by AtnNn** [comment]
A fix for this issue has been released in [RethinkDB 2.3.6](https://rethinkdb.com/blog/2.3.6-release/)
